### DISCUSSION TITLE
Automated Deployment Through Intune

### DISCUSSION THREAD

**Discussion by ivansieder** [AppHouseKitchen/AlDente-Battery_Care_and_Monitoring#1095] [Q&A] [answered]
Is there any way to properly deploy AlDente through Intune (question is mostly related to the fact, that just installing the app manually doesn't yet install the helper, which requires admin permissions)

**Comment by ivansieder** [comment]
@MatthiasKerbl I'm personally using AlDente Pro and I'm absolutely happy with that, however in a corporate environment it's currently quiet tough to automatically install it without user interaction through Intune for example. Is there any guidance on how to do so with AlDente?

**Reply by MatthiasKerbl** [reply]
Hi @ivansieder,

I am glad to hear that you like AlDente. :)
In general, we are offering custom integrations with MDM software such as Intune for business customers. If you are interested in getting AlDente Pro for your company, you can get in contact with us through our support.

Best,
Matthias

**Reply by ivansieder** [reply]
@MatthiasKerbl that sounds awesome! As I don't have the mail of my personal pro version anymore (bought it in February 2023), is there any way you're able to find my email from there and write me directly?

**Reply by MatthiasKerbl** [reply, marked as answer]
You can send us a message to and our support will get in touch with you.

**Reply by ivansieder** [reply]
Done, thanks!
